aten/src/ATen/native/native_functions.yaml

```diff
@@ -14866,6 +14867,7 @@
 - func: _scaled_dot_product_fused_attention_overrideable(Tensor query, Tensor key, Tensor value, Tensor? attn_bias=None, float dropout_p=0.0, bool is_causal=False, bool return_debug_mask=False, *, float? scale=None) -> (Tensor output, Tensor logsumexp, Tensor cum_seq_q, Tensor cum_seq_k, SymInt max_q, SymInt max_k, Tensor philox_seed, Tensor philox_offset, Tensor debug_attn_mask)
   dispatch:
     CompositeExplicitAutograd: _scaled_dot_product_fused_attention_overrideable
+    XPU: _scaled_dot_product_fused_attention_overrideable_xpu
```

Comment:
@drisspg is this the right place to override it?

Reply:
I dont think so - at least not from within core. This is meant as a generic op that can be used to register different backends to through privateuse1

Reply:
@albanD, @drisspg, the original idea intends to avoid adding a new op. Would you be okay with adding an operation like `_scaled_dot_product_mkldnn_attention`, which is similar to `_scaled_dot_product_cudnn_attention`?

Reply:
Confirmed things w/ Alban, It is okay to add the XPU dispatch to any of the available SDPA apis. Which ever one most clostly aligns w/ what you need for forward backwards makes the most sense. 


In that context this change seems fine